ScoutSuite/providers/azure/rules/findings/securitycenter-security-contacts-no-admin-email-notifications.json

```diff
@@ -0,0 +1,10 @@
+{
+    "dashboard_name": "Security contacts",
+    "description": "Sending email to security contact on alert is off",
+    "rationale": "You should set the 'Send email also to subscription owners' to On (CIS 2.19).",
+    "path": "securitycenter.security_contacts.id",
+    "display_path": "securitycenter.security_contacts",
+    "conditions": [ "and",
+        ["securitycenter.security_contacts.id.alerts_to_admins", "equal", "Off"]
```

Comment:
We started converting the on/off enabled/disabled type variables to booleans. Your variable would become `alerts_to_admins_enabled`, and your rule would check `false` instead of `equal` `off`. This makes the data structure more homogeneous by having actual data types instead of magic strings.